runtime/autoload/provider/clipboard.vim

```diff
@@ -55,13 +55,21 @@ endif
 let s:clipboard = {}
 
 function! s:clipboard.get(reg)
-  if s:selections[a:reg].owner > 0
-    return s:selections[a:reg].data
+  let reg = a:reg == '"' ? '+' : a:reg
+  if s:selections[reg].owner > 0
+    return s:selections[reg].data
   end
-  return s:try_cmd(s:paste[a:reg])
+  return s:try_cmd(s:paste[reg])
 endfunction
 
 function! s:clipboard.set(lines, regtype, reg)
+  if a:reg == '"'
+    call s:clipboard.set(a:lines,a:regtype,'+')
+    if s:copy['*'] != s:copy['+']
+      call s:clipboard.set(a:lines,a:regtype,'*')
+    end
+    return 0
```

Comment:
I think I had plain `return` before, but IIRC someone suggested `return 0` in another PR to be explicit. I could change back to `return` (`set` cannot return anythink interesting. Either it succeds or it `echo`es the error, makes no difference to the `ops.c` logic).